agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2812,7 +2733,7 @@ bool backhaul_manager::handle_1905_autoconfiguration_response(ieee1905_1::CmduMe
             return true;
         }
     } else {
-        LOG(ERROR) << "tlvSupportedRole missing - ignoring autconfig search message";
+        LOG(ERROR) << "tlvSupportedRole missing - ignoring autoconfig response message";
```

Comment:
I don't really like unrelated changes in the same commit - they surprise me.

Reply:
Reverted